src/javascript/app/common/form_validation.js

```diff
@@ -93,6 +97,10 @@ const Validation = (() => {
                         });
                     }
                 });
+                if (has_required && $form.find('.indicates-required').length === 0) {
+                    $btn_submit.parent().append($('<p/>', { class: 'hint' })
+                        .append($('<span/>', { class: 'required_field_asterisk no-margin indicates-required', text: '*' })).append($('<span/>', { text: ` ${localize('Indicates required field')}` })));
```

Comment:
the text was never removed from our translation texts, so I don't have to add it again or update translations.